CHANGELOG.md

```diff
@@ -6,6 +6,16 @@ Breaking changes result in a different major. UI changes that might break custom
 
 ## [next-version]
 
+## [0.16.0-rc.1]
+
+### Fixed
+- Internal: Fixed problem on certain versions of Firefox that no longer supported the old version of getUserMedia
+- Internal: Fixed the `tearDown` method to clear the documents and faces currently in the store
+
```

Comment:
I've updated it on `development`. Shall I cherry-pick the commit or are you planning to rebase `development` into this branch @stephencookdev ?